tests/functional/database-management.test.js

```diff
@@ -176,3 +326,19 @@ function noDbExpectedFailureTests() {
         });
     });
 };
+
+
+function calculateTotalSize(keyValueSize, valueSize) {
```

Comment:
This provides no additional value while implying that it might do something more complicated that `arg1 + arg2`, would recommend removing.

Reply:
Removed function